### ISSUE TITLE
Cannot generate tags file with just certain fields.

### ISSUE THREAD

**Issue by roloenusa** [yongkangchen/atom-ctags#115]
The tags file generated includes the signature of the field which makes it ugly and hard to read when i do cmd-r.

```
!_TAG_FILE_FORMAT       2       /extended format; --format=1 will not append ;" to lines/
!_TAG_FILE_SORTED       0       /0=unsorted, 1=sorted, 2=foldcase/
!_TAG_PROGRAM_AUTHOR    Darren Hiebert  /dhiebert@users.sourceforge.net/
!_TAG_PROGRAM_NAME      Exuberant Ctags //
!_TAG_PROGRAM_URL       http://ctags.sourceforge.net    /official site/
!_TAG_PROGRAM_VERSION   5.8     //
purge_soft      /projects/data/mix/build.sh     /^purge_soft()$/;"      function        line:223
purge_bundles   /projects/data/mix/build.sh     /^purge_bundles()$/;"   function        line:234
purge_soft      /projects/data/iso/build.sh     /^purge_soft()$/;"      function        line:214
....
```

I've tried several commands with no success to remove the signature (i.e ` /^purge_soft()$/;"`). I've even modified the code but whatever tags file is generated always contains the signature. Is there an way to disable this? or at the very least, an option to not display the method signature? It's distracting and bloats the screen.

I've tried running with the following command with no success: 

```
/Users/me/.atom/packages/atom-ctags/vendor/ctags-darwin  --options=/Users/me/.atom/packages/atom-ctags/lib/.ctags --fields=+Kn --excmd=p -u -R -f tags /User/me/projects
```

**Comment by yongkangchen** [comment]
You hope not show signature  like `/^purge_soft()$/;"` when press `cmd-r`?

**Comment by roloenusa** [comment]
I think it would be a good option. It cluters my view. The symbol view that ships with atom is really the ideal format. Some people may have use for the signature, but I like simplicity better :)

**Comment by anatooly** [comment]
How manual start ctags generate process in terminal (like Ubuntu) that all work correct?

**Comment by yongkangchen** [comment]
Fixed by f6d6991
